source/style-guide/seo-guidelines.txt

```diff
@@ -0,0 +1,98 @@
+.. _seo-guidelines:
+
+=====================================
+Search Engine Optimization Guidelines
+=====================================
+
+Search Engine Optimization (SEO) considers the actual terms that users 
+enter into search engines (keywords) and employs best practices to 
+improve traffic to web pages from search engines. The following 
+guidance should help you optimize your documentation pages for search.
+
+Titles
+------
+
+Title pages and subsections according to the following SEO best 
```

Comment:
[suggestion, wording]

It wasn't immediately clear to me that we're using "title" as a verb in this sentence. On initial reading I thought "title page" was a noun. Perhaps we can tweak this to something like:

"Use the following SEO best practices for page and subsection titles: "